yarn.lock

```diff
@@ -7368,10 +7368,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/solana-wallet-snap@npm:^2.3.4":
-  version: 2.3.4
-  resolution: "@metamask/solana-wallet-snap@npm:2.3.4"
-  checksum: 10/1c453786e2cded5358d5e53761f094c91007d461399e7f17554bd88be4df680b9420ca74f4da5e92481de64c03897207c7e652985f581f03dd7e42da17be1352
+"@metamask/solana-wallet-snap@npm:^2.3.6":
+  version: 2.3.6
+  resolution: "@metamask/solana-wallet-snap@npm:2.3.6"
+  checksum: 10/84ef57f5c570be398b3f1ea1a9e46dff3404ad3d7f1b0316d26534ef17c5a01a10b152a391457782673ddde4131cfc08750fb6d0dc3fe760c1d250f27521027d
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/35642, which was not included in 13.3.0, but was cherry-picked in 13.2.1.